micro_ros_setup/README.md

```diff
@@ -85,14 +88,79 @@ By running `configure_firmware.sh` command the installed firmware is configured
 ros2 run micro_ros_setup configure_firmware.sh [configuration] [options]
 ```
 
-For NuttX, several different configurations are supported.
-
-| NuttX Configuration | Features |
-|-|-|
-| uros |  Micro-ROS publisher & subscriber, Kobuki demo |
-| drive_base | Kobuki demo |
-| pingpong | Round-Trip-Time test over serial |
-| pingpong-eth | Round-Trip-Time test over Ethernet |
+The following table shows the avaiable configurations for each RTOS/platform.
+For more information please visit the links.
+
+<table>
```

Comment:
why are you using an HTML table instead of a Markdown table?

Reply:
Because it allows combining rows and columns. If you prefer markdown for easier maintainability I could undo the changes.

Reply:
It's okay, I just wondered.